digital-credentials/override-permissions-policy.https.sub.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<meta charset="utf-8" />
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+<script src="/permissions-policy/resources/permissions-policy.js"></script>
+<script src="/common/get-host-info.sub.js"></script>
+<body></body>
+<script>
```

Comment:
What is the difference between this test and enabled-on-self-origin-by-permissions-policy.https.sub.html ?

Reply:
The intent here was to check explicitly overriding the policy through the allow attribute. Here, the policy is disabled by the HTTP header on the top-level.

🤔 ... Now I'm wondering if maybe these should move to digital-credentials/disabled-by-permissions-policy.https.sub.html ... though I'm ok to leave them here.

Reply:
Yeah, same about consistency. Thanks for spotting this! 

I'll fix that up now.